homestead-prov/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM clearwater/base
 MAINTAINER maintainers@projectclearwater.org
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes homestead-prov clearwater-prov-tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes homestead homestead-prov clearwater-prov-tools
```

Comment:
I think this has probably come in from a different set of changes. Would be good to revert this change here to keep the PR clean

Reply:
Ah woops. Yes that's my mistake for not keeping my repos clean and separated. I'll fix that.